### ISSUE TITLE
[Q] Is it possible to run fork twice?

### ISSUE THREAD

**Issue by takisawa** [cookpad/kuroko2#62]
I executed the following Kuroko2 script.
```
### Kuroko2 Script
execute: echo 1
fork:
  execute: echo 2
  execute: echo 3
execute: echo 4
fork:
  execute: echo 5
  execute: echo 6
execute: echo 7

### Result
echo1
  echo 2
  echo 3
echo 4
echo 7
# `echo 5` and `echo 6` were not executed.
```

Is this in the specifications?

**Comment by eisuke** [comment]
No, It is a bug. I'll fix it.
Thank you for reporting the issue.

**Comment by eisuke** [comment]
Fixed this issue, and released as v0.4.2.

**Comment by takisawa** [comment]
Thank you so much!
